go.mod

```diff
@@ -3,8 +3,10 @@ module github.com/prometheus/client_golang
 require (
 	github.com/beorn7/perks v1.0.1
 	github.com/cespare/xxhash/v2 v2.1.2
+	github.com/davecgh/go-spew v1.1.1
 	github.com/golang/protobuf v1.5.2
 	github.com/json-iterator/go v1.1.12
+	github.com/pmezard/go-difflib v1.0.0
```

Comment:
I don't want to be a pain but this module is `THIS PACKAGE IS NO LONGER MAINTAINED.` Not the best, since all users depending on our module will have to download it too.

I wonder if it makes sense to just copy the code for now with the required code (with test) over. Looks reasonably small. There is not benefit in depending on module that does not allow merging new changes. 

Or we could find different package.

Reply:
🙃that was my question previously https://github.com/prometheus/client_golang/pull/998#discussion_r829694057

Ideally, I would prefer an alternative to the package but I couldn't find a good replacement. So maybe coping small piece of code and maintaining it over here ?!!!

Reply:
Yea, I think copy might be better